### PR TITLE
fix issue: #2387 to make tokenizer more robust

### PR DESCRIPTION
**Pull request by wj-Mcat**
### PR types
Bug fixes

### PR changes
Models

### Description
there is a stupid error in my [pr](https://github.com/PaddlePaddle/PaddleNLP/pull/2395), so I create an another pr to fix it.

```python
            elif not os.path.isfile(init_kwargs[args_name],
                                    '') and os.path.isfile(file_path):
```

changed to:

```python
            elif not os.path.isfile(init_kwargs[args_name] or 
                                    '') and os.path.isfile(file_path):
```

### CODE REVIEW

File: examples/text_to_knowledge/wordtag/README.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ WordTag共包含66种词性及专名类别标签，标签集合如下表
             <td>外语单词</td>
         </tr>
         <tr>
-        	<td>组织机构类_体育组织机构_概念</td>
```
Comment (wj-Mcat) [inline examples/text_to_knowledge/wordtag/README.md:148]: When I commit the change,  there is a linting issue in this pr fixed by yapf.

**Review by wj-Mcat — COMMENTED** [review]
Another PR to improve transformer utils.

**Comment by wj-Mcat** [comment]
@guoshengCS please help me review it. It's my fault that pushing a stupid code.

**Comment by guoshengCS** [comment]
注意基于 PaddleNLP 最新develop分支代码checkout新的分支来进行修复，看这里带了很多以前的commit

**Pull request closed without merging**